src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts

```diff
@@ -329,6 +330,7 @@ export class ChargingStationsListTableDataSource extends TableDataSource<Chargin
         new IssuerFilter().getFilterDef(),
         new SiteTableFilter().getFilterDef(),
         new SiteAreaTableFilter().getFilterDef(),
+        new ConnectorTableFilter().getFilterDef()
```

Comment:
Remove from charging station list and add it in Session lists as it is mentioned in the issue name  ;-)

Reply:
My fault ! It's good now.